addons/website/static/src/scss/website.scss

```diff
@@ -341,7 +341,6 @@ table.table_desc tr td {
 // Title
 .s_title {
     .s_title_boxed {
-        display: inline-block;
```

Comment:
12.0 bug so this will be for another branch, but waiting for NMI's opinion on this.

Reply:
ok, let me know if its need to revert back :)

Reply:
@Cocographique ping (https://www.odoo.com/web#id=1943829&action=327&model=project.task&view_type=form&menu_id=4720)

Reply:
What do you think about moving **text-align: center;** to the column?

```
<div class="col-lg-12 s_title" style="text-align: center;" data-name="Title">
    <h1 class="s_title_thin" style="font-size: 62px;">Catchy Headline</h1>
</div>
```

Reply:
Won't that solve the problem for this snippet and leave the problem for all other ones ? (did not check)

Reply:
After further investigation, it seems that it's not as simple as that.
First, I found another bug: 
This line: `<h1 class="s_title_thin" style="font-size: 62px;">Catchy Headline</h1>`
should be: `<h1 class="s_title_thin"><font style="font-size: 62px;">Catchy Headline</font></h1>`
in order to work correctly with the editor.

So the only real solution, is to add a `<span>` and adapt the CSS rule as well. Like this:
```
<h1 class="s_title_thin" style="text-align: center;">
    <span><font style="font-size: 62px;">Catchy Headline</font></span>
</h1>
```

But I'm not a big fan of adding an extra tag just for styling.

Here is the list of the snippets using this option and that are concerned:

- s_title
- s_cover
- s_banner
- s_picture
- s_carousel
- s_comparisons
- s_faq_collapse
- s_parallax